src/components/attachments-list/attachments-list.tsx

```diff
@@ -14,14 +14,15 @@ const {withText} = ui.preacti18n;
 interface AttachmentsListProps extends AssetsListProps {
   files: KalturaAttachmentAsset[];
   downloadPluginManager: DownloadPluginManager;
+  displayFirst: boolean;
```

Comment:
so what does displayFirst do ?

Reply:
displayFirst is holding the information if we this list is the first list that will be shown on the modal, usually we will see: flavors, captions then attachment. but if user disable for example flavors and caption then only attachment will be shown.
In this case checking isDefault is not relevant (relevant only if displayFlavors is enabled) so we need another flag.

Reply:
so if displayFirst is true it will be in a case where only one of the lists is enabled and the other two are not ?

Reply:
it can be also if two lists are enabled, but one that need to be shown before is disabled.
for example: 
- flavors - disabled, caption - enabled, attachment - enabled
- flavors - disabled, caption - disabled, attachment - enabled